learn/validator-trustscore.md

```diff
@@ -1,59 +1,49 @@
 # Nimiq Validator Trustscore
 
-An algorithm that calculates a score to help users assess how reliable a validator is.{.subline .mb-32}
+<Callout type='warning'>
 
-The Validator Trustscore (VTS) algorithm is designed to help users assess the reliability of validators in the Nimiq Wallet. This score ranges from 0 to 1, where 0 indicates a validator is not trustworthy, and 1 indicates a highly trustworthy validator. This way, stakers can make better informed decisions about which validators to trust. The algorithm is based on three key factors:
+ The Validator Trustscore is still under heavy development and nothing is final. Feel free to share your [Suggestions and Feedback](#suggestions-feedback).
```

Comment:
```suggestion
The Validator Trustscore is still under development and we are open to discuss changes to the algorithm.
```

The current copy feels like we are currently changing the algorithm, which is not the case. Maybe we should say that we are open to changing it if we think the feedback is useful?